### ISSUE TITLE
[FR]: Option to NOT to create c.html in home directory on article previewing

### ISSUE THREAD

**Issue by r23vme** [martinrotter/rssguard#1506]
### Brief description of the feature request

When checkbox "Article"->"Enable article preview" is checked selecting articles creates "c.html" file containing corresponding preview. 

A setting to NOT to create such a file or to create it in user specified location, like /tmp/c.html, would be nice.

**Comment by martinrotter** [comment]
How do you compile RSS Guard? How did you install RSS Guard?

File "c.html" is there for debugging purposes and is NOT created in Release builds of RSS Guard, it is only created in Debug build.

**Comment by martinrotter** [comment]
Anyway, I removed the code.

aaed843e4

**Comment by r23vme** [comment]
Thank you!

> How do you compile RSS Guard? How did you install RSS Guard?

I install it from standard gentoo repository. Thats the ebuild with gentoo stuff omitted:

```
...
src_configure() {
	local mycmakeargs=(
		-DBUILD_WITH_QT6=$(usex qt6)
		-DNO_LITE=$(usex webengine)
		-DREVISION_FROM_GIT=OFF
		-DNO_UPDATE_CHECK=ON
		-DENABLE_COMPRESSED_SITEMAP=ON
		-DENABLE_MEDIAPLAYER_QTMULTIMEDIA=$(usex qtmultimedia)
		-DENABLE_MEDIAPLAYER_LIBMPV=$(usex libmpv)
		# recommended
		-DMEDIAPLAYER_FORCE_OPENGL=ON
	)

	cmake_src_configure
}
...
```
https://gitweb.gentoo.org/repo/gentoo.git/tree/net-news/rssguard/rssguard-4.7.3.ebuild

I probably should have risen the issue with the gentoo repo maintainers, sorry to bother you. Will adding "-DCMAKE_BUILD_TYPE="Release"" to mycmakeargs in the ebuild above eliminate the issue without making you to remove debug code?

**Comment by martinrotter** [comment]
> Will adding "-DCMAKE_BUILD_TYPE="Release"" to mycmakeargs in the ebuild above eliminate the issue without making you to remove debug code?

Yes, it will. Moreover I removed the debug line entirely and released 4.7.4 now, so update to 4.7.4 should resolve your problem anyway.
